packages/editor/src/ui-tools/viewport/di.config.ts

```diff
@@ -39,10 +36,6 @@ const ivyViewportModule = new FeatureModule(
     configureCommand(context, SetViewportCommand);
     configureCommand(context, RepositionCommand);
 
-    bindAsService(context, TYPES.IDiagramStartup, RestoreViewportHandler);
-    configureActionHandler(context, EnableDefaultToolsAction.KIND, RestoreViewportHandler);
-    configureActionHandler(context, FocusDomAction.KIND, RestoreViewportHandler);
```

Comment:
`RestoreViewportHandler` seems to have a handle condition that is never fulfilled